### PR TITLE
Adding ESLint to app dev dependencies.

### LINKED ISSUE #424: Add ESLint and plugins to App devDependencies
### What

Add required ESLint dependencies to app devDependencies to ensure linting feedback is provided correctly.

You should be able to use the ESLint output in VSCode to see if it's working and what packages need to be added.

### Why

Ensure linting feedback for all developers

### When

ASAP

### PR DESCRIPTION
**Pull request by jasonvalenzuela**
Resolves #424. Adds linting feedback to developers and displays feedback in IDE (checked on VSCode). Removes package-lock.json file, which was displaying a warning message when building/running the project.

### CODE REVIEW

File: app/.eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
```
Comment (jaronheard) [inline app/.eslintrc.json]: There's a `.eslintrc` file as well already in the repo, which just has the settings from `civic`. Does this supplement or replace it?

Comment (jaronheard) [reply app/.eslintrc.json]: Looks like it replaces it. That could be fine, but in that case the other file should be removed.

**Review by jaronheard — CHANGES_REQUESTED** [review]

**Comment by jaronheard** [comment]
@jasonvalenzuela Looks like the tests are failing -- it's possible that this because the integrity checks are being stripped from the `yarn.lock` file?

I think this issue will be fixed by upgrading your yarn version. Even better, we can check in a version of yarn using `yarn-path` as described here to avoid this issue in the future. More info: https://github.com/yarnpkg/yarn/issues/6440

File: app/.eslintrc.json (hunk 1 above)
Comment (jasonvalenzuela) [reply app/.eslintrc.json]: I didn't see the `.eslintrc` in the repo! I'll use that file instead to keep everything consistent.

**Comment by jasonvalenzuela** [comment]
Will open a new PR for this.

**Pull request closed without merging**